.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+
+New Feature
+
+- [ ] Clean code (small functions, proper variable names, PEP8 etc)
```

Comment:
Worth mentioning the style guide https://github.com/fastats/fastats/blob/c7127198686464f5906ffbf04327bf1e565fde4e/CONTRIBUTING.md#code-style